catalog/app/components/MIcon/index.js

```diff
@@ -33,7 +33,7 @@ const Icon = styled(mapProps(omit(['spin']))(FontIcon))`
 `;
 
 // eslint-disable-next-line object-curly-newline
```

Comment:
https://eslint.org/docs/rules/object-curly-newline
we could decrease the amount of eslint poop considerably with a more reasonable setting, e.g. consistent=true